app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,7 @@ def new
       flash[:notice] = "You are already logged in"
       if user.admin_user?
         redirect_to "/admin"
```

Comment:
A minor thing => can we change this to `admin_path` just to practice using a path helper?